modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -0,0 +1,95 @@
+
+----
+  labels:
+    rtf.mulesoft.com/synchronized: "true"
+----
+
+* If you need to authenticate access to your local registry, ensure that the https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials[corresponding credentials are configured as a Kubernetes secret in the `rtf-` namespace^]. You'll need the name of the configured secret when you install Runtime Fabric. 
+
+* You also need to synchronize in your registry the Mule runtime versions you intend to use for application deployments. Refer to the xref:release-notes::runtime-fabric/runtime-fabric-runtimes-release-notes.adoc[Runtime Fabric Patch Update Release Notes] for information about the latest Mule runtime patch versions. 
+
+== Configure a Local Registry for Use with Runtime Fabric 
+
+To configure a local registry, you must gather and add the necessary credentials to synchronize the registries. 
+
+=== Before You Begin
+
+Ensure that you've performed the following tasks: 
+
+. Set up, configure, and test a local Docker image registry.
+. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
++
+See the xref:release-notes::runtime-fabric/runtime-fabric-release-notes.adoc#1120[Runtime Fabric release notes] for the required dependencies for your Runtime Fabric version. 
++
+[NOTE]
+Dependency versions are specific to the Runtime Fabric version.
+ 
+
+. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Download and install the `rtfctl` command line utility] version 0.3.150 or higher.
+. xref:install-self-managed.adoc#step-3-download-the-rtfctl-utility[Create a Runtime Fabric using Runtime Manager], and gather the activation data. 
+
+=== Configure Your Local Registry
+
+. Obtain `rtf-runtime-registry` credentials:
+
+----
+# rtfctl get docker-registry-login <activation-data>
+----
+
+This command sets `RTF_DOCKER_SERVER`, `RTF_DOCKER_USERNAME`, and `RTF_DOCKER_PASSWORD` in the current shell environment. 
+
+. Run the following commands to verify that the Docker login to the `rtf-runtime-registry` succeeds:
++
+---- 
+# eval $(rtfctl get docker-registry-login)
+# docker login $RTF_DOCKER_SERVER --<username> $RTF_DOCKER_USERNAME --<password> $RTF_DOCKER_PASSWORD
+----
+
+. To pull and synchronize images, run the following command, replacing the `<username>` and `<password>` values where appropriate:
++
+---- 
+# docker login $RTF_DOCKER_SERVER --<username> $RTF_DOCKER_USERNAME --<password> $RTF_DOCKER_PASSWORD → docker pull rtf-runtime-registry.kqa.msap.io/mulesoft/rtf-agent:v1.12.0 → docker tag rtf-runtime-registry.kqa.msap.io/mulesoft/rtf-agent:v1.12.0 localhost:5000/mulesoft/rtf-agent:v1.12.0 → docker push localhost:5000/mulesoft/rtf-agent:v1.12.0
```

Comment:
Are we going to change this to prod URLs?

Reply:
@rohit-verma7 Good catch, we should change this to whatever is accurate